### DISCUSSION TITLE
Radius of gyration for FLIGHTLAB/RCAS

### DISCUSSION THREAD

**Discussion by rhig2000** [wenbinyugroup/.github#35] [VABS Related Discussions]
FLIGHTLAB requires Blade Section Mass Radius of Gyration about CG, RoG Y CG (ft), RoG Z CG (ft), RoG YZ CG (ft), presumably similar to RCAS. I expect these values can be pulled directly from the 6X6 Mass Matrix in VABS output. Which values correspond? And are the values given in units of length?

**Comment by wenbinyu-PU** [comment]
I don't know the exact definition of RoG in Flightlab. However, they should be given in the units of length. VABS outputs RoG defined as the square root of the mass moment of inertia about x1 divided by the mass per unit length
